src/main/java/net/pms/formats/v2/AudioAttribute.java

```diff
@@ -1,5 +1,5 @@
 /*
- * PS3 Media Server, for streaming any media to your PS3.
+ * PS3 Media Server, for streaming any medias to your PS3.
```

Comment:
That rules / logic look weird to me. It's just out of my understanding.
I'm naive to think that when we see mistake(s) we try to correct them.

So i'm not surprise if we still have our fake democratic societies and corrupt politicians for again long...